test/Codenizer.HttpClient.Testable.Tests.Unit/WhenMatchingRoutes.cs

```diff
@@ -64,6 +72,53 @@ public void GivenNonConfigured_NoResultIsReturned()
                 .Should()
                 .BeNull();
         }
+        
+        
+        [Fact]
+        public void GivenRouteWithHeaders_RequestBuilderIsReturned()
```

Comment:
Could you change the test name to `GivenRouteWithAuthorizationHeader_RequestBuilderIsReturned()`?
It's no longer _any_ header but a specific one and that would help with communicating what this test is about.